### PR TITLE
fix vi_mode set but still not enabled in prompt_tookit shell

### PR DESCRIPTION
**Pull request by tsingakbar**
As the title suggests.

### CODE REVIEW

**Comment by scopatz** [comment]
Thanks for putting this in @tsingakbar!

File: xonsh/ptk/shell.py [hunk 1]
```diff
@@ -77,7 +77,8 @@ def singleline(self, store_in_history=True, auto_suggest=None,
                     enable_history_search=enable_history_search,
                     reserve_space_for_menu=0,
                     key_bindings_registry=self.key_bindings_manager.registry,
-                    display_completions_in_columns=multicolumn)
+                    display_completions_in_columns=multicolumn,
+                    vi_mode=builtins.__xonsh_env__.get('VI_MODE'))
```
Comment (scopatz) [inline xonsh/ptk/shell.py]: Can you make this a little more robust, please, by testing the `xonsh.shell.prompt_toolkit_version()` to figure out which kwarg to use? That would be great!

File: xonsh/ptk/shell.py [hunk 2]
```diff
@@ -36,7 +36,7 @@ def __init__(self, **kwargs):
             enable_auto_suggest_bindings=True,
             enable_search=True,
             enable_abort_and_exit_bindings=True,
-            enable_vi_mode=Condition(lambda cli: builtins.__xonsh_env__.get('VI_MODE')),
+            # enable_vi_mode=Condition(lambda cli: builtins.__xonsh_env__.get('VI_MODE')), # deprecated acoording to prompt_toolset 1.0 document
```
Comment (scopatz) [inline xonsh/ptk/shell.py]: Also, please don't leave in commented out code lines.

File: xonsh/ptk/shell.py (hunk 1 above)
Comment (tsingakbar) [inline xonsh/ptk/shell.py]: ok

File: xonsh/ptk/shell.py (hunk 2 above)
Comment (tsingakbar) [inline xonsh/ptk/shell.py]: ok

File: xonsh/ptk/shell.py [hunk 3]
```diff
@@ -32,12 +32,22 @@ def __init__(self, **kwargs):
...
+
+        key_bindings_manager_args = {
+                'enable_auto_suggest_bindings': True,
+                'enable_search': True,
+                'enable_abort_and_exit_bindings': True,
+                'enable_open_in_editor': True
+                }
+        import prompt_toolkit
+        vptk = getattr(prompt_toolkit, '__version__', '0.0')
+        major, minor = [int(x) for x in vptk.split('.')[:2]]
```
Comment (scopatz) [inline xonsh/ptk/shell.py]: Would you mind using `xonsh.shell.prompt_toolkit_version()` function, please? This duplicates that code to some degree. Thanks!

Comment (tsingakbar) [inline xonsh/ptk/shell.py]: sure, previously I just felt a little confused about how to call that function.

**Comment by scopatz** [comment]
Hi @tsingakbar - sorry I didn't notice that you had updated this.  Please comment in the issue so that we get an email.  Thanks again!

**Pull request merged**